library/src/webapp/bundled-js/sakai.morpheus.reorderer.js

```diff
@@ -106,15 +106,19 @@ $PBJQ(document).ready(function(){
     });
 
     // the jquery-ui sortable initialization
-    return $PBJQ('[id$="reorder-list"]').sortable({
-      items: '.reorder-element:not(.notsortable)',
-      start: function( event, ui ) {
-        preserveStatus(ui);
-      },
-      update: function( event, ui ) {
-        registerChange(event, ui);
-      },
-    });
+    if ($PBJQ('[id$="reorder-list"]').length) {
+        return $PBJQ('[id$="reorder-list"]').sortable({
+            items: '.reorder-element:not(.notsortable)',
+            start: function( event, ui ) {
+            preserveStatus(ui);
+            },
+            update: function( event, ui ) {
+            registerChange(event, ui);
+            },
+        });
+    } else {
+        return null;

```

Comment:
What happens when you return null from here?

Reply:
no error, because it won't execute it unless that id is present.

Reply:
If the statement makes sure not to run .sortable() if the id is not present in the markup, therefore no error.